api-specificatie/brc/openapi.yaml

```diff
@@ -2271,6 +2432,12 @@ components:
           - ztc
           - drc
           - brc
+        requestId:
+          title: Request id
```

Comment:
Dit Request id lijkt niet te passen bij de overige wijzigingen die allemaal over het toevoegen van ETags gaan. Is dit misschien een restje van een 1.1 wijziging? Zo niet, wat betekent dit Request id dan?

Reply:
@michielverhoef dit hoort bij de audittrail functionaliteit en kan samenwerken met NLX.

Als deze in de diff zit/zat, dan denk ik dat het komt doordat er een schema-generatie vergeten was een paar weken terug toen audittrail aangepast werd.